tests/conftest.py

```diff
@@ -66,18 +66,25 @@ def limited_rate():
     def unlimited_rate():
         return 'test'
 
-    Limiter(
-            app_,
-            key_func=obj_or_import_string(
-                app_.config.get('RATELIMIT_KEY_FUNC'),
-                default=useragent_and_ip_limit_key)
-        )
     return app_
 
 
+@pytest.fixture()
+def no_limiter_app(base_app):
```

Comment:
Maybe it would be better if the naming was reversed "app" and "app_with_limiter".